src/AWS/CRT/Internal/FFI.php

```diff
@@ -12,6 +12,7 @@
 final class FFI {
     private static $ffi = null;
     private static $refcount = 0;
+    private static $NULL = null;
```

Comment:
why is this capitalized?

Reply:
because in C, NULL is capitalized, and it doesn't matter because this is all crap.